client/features/output-visualization-page/components/UndoAndRedo.tsx

```diff
@@ -3,6 +3,7 @@ import ButtonGroup from '@mui/material/ButtonGroup';
 import IconButton from '@mui/material/IconButton';
 import UndoRoundedIcon from '@mui/icons-material/UndoRounded';
 import RedoRoundedIcon from '@mui/icons-material/RedoRounded';
+import styles from '@styles/UndoAndRedo.module.css';
 
 import { useStoreWithEqualityFn } from 'zustand/traditional';
```

Comment:
This zustand store might have been overlooked in a previous PR. Could you help enhance it by:
- Using a more descriptive name, such as `useUndoRedoStore` for clarity
- Moving the store to the `/stores` folder

Reply:
Thanks for the review! All suggestions have been implemented.